docs/api/assets/css/main.css

```diff
@@ -492,6 +492,8 @@ ul.tsd-descriptions > li > :last-child > :last-child > :last-child {
   padding-bottom: 200px; }
 
 .row {
+  display: -ms-flexbox;
+  display: flex;
```

Comment:
Interesting. This is because the theme changed. I'll look into getting the tests to fail if the docs have changed but not been committed, so we can avoid this.